webapp/app/statisticalFactsheets/ForestArea/ForestArea.js

```diff
@@ -1,12 +1,39 @@
 import './forestArea.less'
-import React from 'react'
+import React, { useEffect } from 'react'
+import PropTypes from 'prop-types'
+import { Bar } from 'react-chartjs-2'
+
+import { useGetRequest, useI18n } from '@webapp/components/hooks'
+
+import * as APIUtils from '../utils/apiUtils'
+import * as ChartUtils from '../utils/chartUtils'
+import { getParams } from '../utils/apiUtils'
+
+const ForestArea = (props) => {
+  const { levelIso } = props
+  const i18n = useI18n()
+  const section = 'forestArea'
+  const chartHeads = ['1990', '2000', '2010', '2020']
+
+  const url = APIUtils.getUrl(levelIso)
+  const params = getParams(section)
+
+  const { data, dispatch: fetchData, loaded } = useGetRequest(url, {
+    params,
+  })
+
+  useEffect(fetchData, [url])
```

Comment:
this is somehow similar to all components. can we have a generic hook?

Reply:
I thought so too, will do